flask_combo_jsonapi/decorators.py

```diff
@@ -73,8 +73,12 @@ def wrapper(*args, **kwargs):
         try:
             return func(*args, **kwargs)
         except JsonApiException as e:
+            try:
+                status = int(e.status)
+            except:
```

Comment:
hi! could you please provide some exception type here? we don't want a bare except, or `Exception`

Reply:
Restricted to `ValueError` in ad22482c58028efd182eecc6d5f5d67265600619